blog/_posts/2024-08-07-eks-to-aks-series-part1.md

```diff
@@ -0,0 +1,33 @@
+---
+title: "Mastering the Move: EKS to AKS by Example"
+description: "Learn how to migrate Amazon EKS workloads to Azure AKS using KEDA and Karpenter in this comprehensive guide."
+date: 2050-08-24
+author: Kenneth Kilty
+categories: general
+---
+
+## Introduction
+
+Many companies use multiple clouds for their workloads. These companies need to accommodate the cloud preferences of their customers. Kubernetes plays a central role in multi-cloud workloads due to its ability to provide a consistent and portable environment across different cloud providers.
+
+We’re excited to announce the first in a new documentation series designed specifically for customers already using Amazon EKS, to help them replicate or migrate their workloads to AKS: [Replicate an AWS event-driven workflow (EDW) workload with KEDA and Karpenter in Azure Kubernetes Service (AKS)](https://learn.microsoft.com/en-us/azure/aks/eks-edw-overview)
+
+Moving a workload from another managed Kubernetes offering to AKS can be challenging. This series aims to provide a holistic, end-to-end experience by example to help ease that process. The samples in this series begin with infrastructure and code on EKS and end with equivalently functional infrastructure and code on AKS, while explaining the conceptual differences between AWS and Azure throughout.
+
+## EKS to AKS - Event Driven Workflow Scenario
+
+This new [article](https://learn.microsoft.com/en-us/azure/aks/eks-edw-overview) guides you through replicating an Amazon Web Services (AWS) Elastic Kubernetes Service (EKS) event-driven workflow (EDW) workload using [KEDA](https://keda.sh/) and [Karpenter](https://karpenter.sh/) in Azure Kubernetes Service (AKS). In the spirit keeping the workload as portable as possible, this guide prioritizes minimizing the changes needed to get the workload running on Azure.
```

Comment:
On the point about minimizing changes - it sounds like there's an implicit tradeoff there that you might want to call out.  Is there some way in which the architecture could be even better if you leaned into azure fully? That could be a good thing to mention as it suggests that there's a spectrum of portability to value that the user can move along based on their goals.